src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="age"
+          >
+        </label>
+
+        <label class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="16"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="answer"
+            value="yes"
+          >
+          Yes
```

Comment:
Text is also considered to be new level of nesting, so it should have 2 indents as well
```suggestion
            Yes
```